linker/Linker.Steps/MarkStep.cs

```diff
@@ -811,6 +835,34 @@ bool ProcessLateMarkedAttributes ()
 			return markOccurred;
 		}
 
+		bool ProcessOnlyIfInstantiated ()
+		{
+			// Certain things on a type only need to be marked if an instance of the type could be instantiated
+			var startingQueueCount = _onlyIfInstantiated.Count;
```

Comment:
This variable seems to be redundant

Reply:
I was able to eliminate this entire method.